tools/scoring-api.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/python3
+#
+# Usage:   ./tools/scoring-api.py MODEL_NAME DATASET.CSV WEIGHTSFILE.H5 [PORT] [PARAM=VALUE]...
+# Example: ./tools/scoring-api.py rnn data/anssel/yodaqa/full-dataset.csv weights/weights-rnn-38a84de439de6337-bestval.h5 5001 loss=\"binary_crossentropy\"
+# This script loads keras model and pre-trained weights for sentence/property selection
+# and starts REST API for scoring question - sentence pairs.
+# The script listens on given or default (5000) port and accepts JSON in following format:
+#    {"question":"QUESTION_TEXT","labels":["SENTENCE1", "SENTENCE2", ...]}
+# Example:
+#    {"question":"what country is the grand bahama island in","labels":["contained by", "contained in", "contained"]}
+# The response is JSON object with key "score" and value containing list of scores, one for each label given
+#    {"score":[SCORE1, SCORE2, ...]}
```

Comment:
Sorry for the nitpicking, but can you make the header look a little bit more like all the other scripts, i.e. short summary at top, then usage + example, and parapgraphs separated by empty comment lines? Thanks!